src/components/Player.vue

```diff
@@ -124,29 +132,35 @@
             <li @click="changeName">
               <font-awesome-icon icon="user-edit" />Rename
             </li>
-            <li v-if="!session.nomination" @click="nominatePlayer()">
-              <font-awesome-icon icon="hand-point-right" />
-              Nomination
-            </li>
-            <li @click="movePlayer()">
-              <font-awesome-icon icon="redo-alt" />
-              Move player
-            </li>
-            <li @click="swapPlayer()">
-              <font-awesome-icon icon="exchange-alt" />
-              Swap seats
+            <li v-if="!session.nomination" @click="removePlayer">
+              <font-awesome-icon icon="times-circle" />
+              Remove
             </li>
+            <template v-if="!session.nomination">
+              <li @click="movePlayer()">
+                <font-awesome-icon icon="redo-alt" />
+                Move player
+              </li>
+              <li @click="swapPlayer()">
+                <font-awesome-icon icon="exchange-alt" />
+                Swap seats
+              </li>
+              <li @click="toggleOnBlock()">
+                <font-awesome-icon icon="skull" />
+                {{ player.isOnBlock ? "Take off block" : "Put on block" }}
+              </li>
+              <li @click="nominatePlayer()">
+                <font-awesome-icon icon="hand-point-right" />
+                Nomination
+              </li>
+            </template>
             <li
               @click="updatePlayer('id', '', true)"
               v-if="player.id && session.sessionId"
             >
               <font-awesome-icon icon="chair" />
               Empty seat
             </li>
-            <li @click="removePlayer">
-              <font-awesome-icon icon="times-circle" />
-              Remove
```

Comment:
Because the "nomination" item is (by far) the most used item on that menu, so making it the one that is also right next to the player name is helpful + less misclicks.

Reply:
Alright, I'd still prefer to keep the actions that remove something (the player, the seat claim) near the bottom. So maybe only move the nomination there.